.travis.yml

```diff
@@ -278,8 +278,8 @@ matrix:
         - sed -n '/``` cpp/,/```/{/```$/Q;/```/d;p;}' ${EVENTS}/README.md > main.cpp
         - |
           rm -r rtos/source/TARGET_CORTEX drivers/source/usb connectivity/cellular connectivity/drivers/cellular \
-            connectivity/libraries/ppp connectivity/netsocket connectivity/nanostack connectivity/lwipstack features/frameworks/greentea-client \
-            features/frameworks/utest features/frameworks/unity components BUILD
+            connectivity/libraries/ppp connectivity/netsocket connectivity/nanostack connectivity/lwipstack \
+            components BUILD
```

Comment:
Question: Why `features/frameworks/greentea-client  features/frameworks/utest features/frameworks/unity`  was removed instead of update with `TESTS/COMMON/xx` new path

Reply:
This is for the bare metal test in CI, which builds an application with RTOS-required libraries removed. `TESTS` is excluded from compilation anyway, so it doesn't have to be removed.